package.json

```diff
@@ -25,8 +25,10 @@
     "prettier-plugin-svelte": "^3.3.2",
     "svelte": "^5.0.0",
     "svelte-check": "^4.0.0",
+    "svelte-turnstile": "^0.9.0",
     "typescript": "^5.7.2",
     "typescript-eslint": "^8.17.0",
-    "vite": "^5.0.3"
+    "vite": "^5.0.3",
+    "lucide-svelte": "^0.468.0"
```

Comment:
this is tree-shakeable and efficient to just use an icon from where only that icon is imported. I'm just using their loading indicator and got lazy but if needed i can make my own svg if we think it matters